File: Baekjoon_Online_Judge/NO_02693/2693.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 08                 | #
# | Day 02                   | #
# | 2693 N번째 큰 수           | #
# ---------------------------- #


if __name__ == "__main__":
    T = int(r_input())

    for _ in range(T):
        print(sorted(map(int, r_input().split()))[-3])

File: Baekjoon_Online_Judge/NO_02693/2693.cpp
#include <iostream>
#include <algorithm>
using namespace std;
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 08                 | 
 | Day 02                   | 
 | 2693 N번째 큰 수           | 
 ---------------------------- 
*/

int main(){
    ios_base::sync_with_stdio(0);
    cin.tie(0);
    cout.tie(0);

    int T;

    cin >> T;

    for(int i = 0; i < T; i ++){
        int A[10];

        for(int i = 0; i < 10; i ++){
            int num;
            cin >> num;

            A[i] = num;
        }

        sort(A, A + 10);

        cout << A[7] << "\n";
    }
    return 0;
}
